features/netsocket/CellularInterface.h

```diff
@@ -115,15 +115,6 @@ class CellularInterface: public NetworkInterface {
      *                  or null if no network mask has been received.
      */
     virtual const char *get_gateway() = 0;
-
-    /** @copydoc NetworkInterface::cellularBase
-     */
-    MBED_DEPRECATED_SINCE("mbed-os-5.12", "Migrated to CellularInterface")
```

Comment:
@SeppoTakalo do we have deprecated documented  - when it can be removed - I can't find the public docs reference.

In this case, 5.12 -> 5.13 does not look sufficient time to remove this one, does it?

Reply:
As the original Deprecation policy allowed us to remove stuff when major version changed, and due the marketing reasons, we are now claiming that 6.0 does not introduce API changes, we are forced to remove deprecated APIs now, before we enter any LTS branch.